suzieq/gui/stlit/xplore.py

```diff
@@ -349,6 +384,17 @@ def _draw_aggrid_df(self, df) -> AgGrid:
         jscode = self._aggrid_style_rows()
         gridOptions['getRowStyle'] = jscode
 
+        start_row = self._state.start_row
+        end_row = self._state.end_row
+        if end_row > df.shape[0]:
+            end_row = df.shape[0]
+
+        if end_row - start_row > MAX_ROW_WARN:
+            st.warning('Printing too many rows can fail or be very slow')
+
+        if (end_row - start_row) < df.shape[0]:
+            st.info(f'Displaying rows {start_row} to {end_row}')
```

Comment:
```suggestion
            st.info(f'Displaying rows {start_row} to {end_row} out of {df.shape[0]}')
```
Knowing the size of the table might be useful when users select the rows to show